core/templates/dev/head/services/UserService.ts

```diff
@@ -12,53 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+require('services/contextual/UrlService.ts');
```

Comment:
Add `require()` for `UserInfoObjectFactory`

Reply:
Okay